src/osd/scrubber/pg_scrubber.cc

```diff
@@ -2029,30 +2050,52 @@ void PgScrubber::dump_active_scrubber(ceph::Formatter* f, bool is_deep) const
     }
     f->close_section();
   }
-  f->dump_string("schedule", "scrubbing");
+  if (m_scrub_job->blocked) {
+    f->dump_string("schedule", "blocked");
+  } else {
+    f->dump_string("schedule", "scrubbing");
+  }
 }
 
 pg_scrubbing_status_t PgScrubber::get_schedule() const
 {
-  dout(25) << __func__ << dendl;
-
   if (!m_scrub_job) {
     return pg_scrubbing_status_t{};
   }
 
+  dout(20 /*25*/) << fmt::format("{}: active:{} blocked:{}",
```

Comment:
Is the extra 25 comment meant to be there?

Reply:
Thanks a lot. All 3 are indeed errors.